src/index.js

```diff
@@ -1,29 +1,153 @@
 import './style.css';
 
-const listArray = [{
-  description: 'wash the dishes',
-  complete: true,
-  index: 0,
-},
-{
-  description: 'complet To Do list project',
-  complete: false,
-  index: 1,
-},
-];
+const textInput = document.querySelector('input');
+const listToDo = document.querySelector('.listToDo');
+class MyObject {
+  constructor(description, completed, index) {
+    this.description = description;
```

Comment:
- [ ] Please, consider fixing the following requirement:

- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

In your case, the indexes are not updating. Please, have a look at the following screenshots.

> This is I added 3 tasks, and you see the indexes are correct in the beginning:
<img width="1113" alt="Screen Shot 2022-08-19 at 09 30 53" src="https://user-images.githubusercontent.com/49064106/185550167-94d0c922-b914-46a7-a25c-fb51d7d41e08.png">

> After, I deleted the first task and the indexes did not update:
<img width="1120" alt="Screen Shot 2022-08-19 at 09 31 22" src="https://user-images.githubusercontent.com/49064106/185550243-7e88d717-54d4-4e90-a0cb-d6fb654c4edb.png">